graph/nullable.md

```diff
@@ -0,0 +1,311 @@
+# Nullable Properties
+
+A nullable property means *only* that the property may have `null` as a value; the "nullability" of a property does not say anything about how a value is set into a property.
+For example, a non-nullable property is *not* required to create a new instance of an entity.
+It only means that the property will have a value when it is retrieved.
+In the case that no value is provided when the entity is created, this means that the service will create one; this value can be specified with the `DefaultValue` attribute, but if the value is contextual and determine at request time, then the property can both be non-nullable *and* have no `DefaultValue` specified. 
+Below are some examples of nullable and non-nullable properties. 
+
+## CSDL
+
+```xml
+<EntitySet Name="servicePrincipals" Type="self.servicePrincipal" />
+...
+<EntityType Name="servicePrincipal">
+  <Key>
+    <PropertyRef Name="id" />
+  </Key>
+  <Property Name="id" Type="Edm.String" Nullable="false" />
```

Comment:
You can resolve when you see this.
I just had an API owner try to use a defaultValue for a primary key...